src/styles/colors.ts

```diff
@@ -0,0 +1,44 @@
+import { ThemesEnum } from '../interfaces/theme.enum';
+
+export const getColors = (theme: ThemesEnum): Colors => ({
+  ...baseColors,
+  ...(theme === ThemesEnum.light && darkGray)
```

Comment:
something with naming or logic.  
I suppose `darkGray` is `darkColors`, but in this case another condition should be there
```
...(theme === ThemesEnum.dark && darkColors)
```